spec/factories/tests.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :test do
+    mission_id 1
+    test_name "MyString"
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.